src/ProcessManager.php

```diff
@@ -658,10 +662,30 @@ protected function commandFiles(array $data, ConnectionInterface $conn)
         try {
             $slave = $this->slaves->getByConnection($conn);
 
+            $start = microtime(true);
+
+            clearstatcache();
+
+            $newFilesCount = 0;
+            foreach (array_diff($data['files'], array_keys($this->filesLastMTime)) as $filePath) {
```

Comment:
I personally don't like that style. Could you please explode this magic line into several?

Reply:
done